docs/how-to/how-to-run-kata-containers-with-SE-VMs.md

```diff
@@ -0,0 +1,387 @@
+
+... log skipped ...
+
+[    1.467228] Run /init as init process
+{"msg":"baremount source=\"proc\", dest=\"/proc\", fs_type=\"proc\", options=\"\", flags=MS_NOSUID | MS_NODEV | MS_NOEXEC","level":"INFO","ts":"2023-06-07T10:17:23.537542429Z","pid":"1","subsystem":"baremount","name":"kata-agent","source":"agent
+","version":"0.1.0"}
+
+... log skipped ...
+
+$ # Press ctrl + a + x to exit
+```
+
+If the hypervisor log does not indicate any errors, it provides assurance that the image
+has been successfully loaded, and a Virtual Machine (VM) initiated by the kata runtime
+will function properly.
+
+Let us proceed with the final verification by running a test container in a Kubernetes
+cluster. Please make user you have a running cluster like:
+
+```
+$ kubectl get node
+NAME           STATUS   ROLES                  AGE     VERSION
+test-cluster   Ready    control-plane,master   7m28s   v1.23.1
+```
+
+Please execute the following command to run a container:
+
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-kata
+spec:
+  runtimeClassName: kata-qemu
+  containers:
+  - name: nginx
+    image: nginx
+EOF
+pod/nginx-kata created
+$ kubectl get po
+NAME         READY   STATUS    RESTARTS   AGE
+nginx-kata   1/1     Running   0          29s
+$ kubectl get po -oyaml | grep "runtimeClassName:"
+    runtimeClassName: kata-qemu
+$ # Please make sure if confidential-guest-support is set and a secure image is used
+$ $ ps -ef | grep qemu | grep -v grep
+root       76972   76959  0 13:40 ?        00:00:02 /opt/kata/bin/qemu-system-s390x
+... qemu arguments ...
+-machine s390-ccw-virtio,accel=kvm,confidential-guest-support=pv0
+... qemu arguments ...
+-kernel /opt/kata/share/kata-containers/kata-containers-se.img
+... qemu arguments ...
+```
+
+Finally, an operational kata container with IBM Secure Execution is now running.
+
+## Using Kata-Deploy with Confidential Containers Operator
+
+It is reasonable to expect that the manual steps mentioned above can be easily executed.
+Typically, you can use
+[kata-deploy](https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/README.md)
+to install Kata Containers on a Kubernetes cluster. However, when leveraging IBM Secure Execution,
+you need to employ the confidential container's
+[operator](https://github.com/confidential-containers/operator).
+During this process, a `kata-deploy` container image serves as a payload image in a custom
+resource `ccruntime` for confidential containers, enabling the operator to install Kata
+binary artifacts such as kernel, shim-v2, and more.
+
+In this section, we will specifically guide you on building a payload image
```

Comment:
minor: remove "we" and use "you will learn" or "This section explains how" / "This section guides you through....."

Reply:
Thanks for the feedback. The document has been updated.